### ISSUE TITLE
Random disconnects

### ISSUE THREAD

**Issue by kaiserdj** [Sopur/Discord-user-bots#52]
I'm getting this error, but the problem is that it doesn't reconnect

```
2023-08-10T04:46:38: Error: WebSocket is not open: readyState 0 (CONNECTING)
2023-08-10T04:46:38:     at WebSocket.send (/home/ubuntu/---/node_modules/ws/lib/websocket.js:442:13)
2023-08-10T04:46:38:     at Timeout._onTimeout (/home/ubuntu/---/node_modules/discord-user-bots/src/client/client.js:84:37)
2023-08-10T04:46:38:     at listOnTimeout (node:internal/timers:564:17)
2023-08-10T04:46:38:     at process.processTimers (node:internal/timers:507:7)
```

**Comment by mboukour** [comment]
Hey, did you find a solution

**Comment by HadiZiady** [comment]
i'm trying to wrap client.on.ready inside try and catch and gonna see if it will solve the issue

**Comment by Sopur** [comment]
Fixed this problem in the 2.0.0 update.
